### PR TITLE
Release v3.3.1

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt [hunk 1]
```diff
@@ -89,51 +89,54 @@ class OverrideForkPointDialog(
...
+              }
             }
-          }
-        ).enabledIf(rb.selected).bindItem(
-          MutableProperty(::customCommit) { customCommit = it }
-        )
+          ).enabledIf(rb.selected).bindItem(
+            MutableProperty(::customCommit) { customCommit = it }
+          )
+        }
```
Comment (PawelLipski) [inline frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt:139]: Whoops this is... suspicious, this closing brace has moved 🤔

Comment (PawelLipski) [reply frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt:139]: Ha! no, still `bind` is call on `buttonsGroupCompat { ... }`, and `enabledIf` is called on `comboBox(...)`.

Anyway, lemme refactor it a bit for clarity...

Comment (PawelLipski) [reply frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt:139]: PR #1444

**Review by mkondratek — DISMISSED** [review]
this is fine

**Pull request merged**